KITE-AppRTC-Test/src/main/java/org/webrtc/kite/apprtc/checks/RemoteVideoHDCheck.java

```diff
@@ -37,7 +38,9 @@ public String stepDescription() {
   
   @Override
   protected void step() throws KiteTestException {
-    final AppRTCMeetingPage appRTCMeetingPage = new AppRTCMeetingPage(webDriver, logger);
+    if (appRTCMeetingPage == null) {
```

Comment:
why do we need this?

Reply:
because of the Logger object, which isn't passed in the Step constructor, hence this workaround.
We should pass it in the constructor instead, but that's a massive change which i decided to do later